lib/private/Server.php

```diff
@@ -410,7 +410,8 @@ public function __construct($webRoot, \OC\Config $config) {
 
 			$previewConnector = new \OC\Preview\WatcherConnector(
 				$root,
-				$c->get(SystemConfig::class)
+				$c->get(SystemConfig::class),
+				$this->get(IEventDispatcher::class)
```

Comment:
```suggestion
				$c->get(IEventDispatcher::class)
```
Other lines from the same file seem to use `$c` for this, but not sure what’s the difference :thinking:

Reply:
I didn't go into the nuances either, but before this initialization, when creating root, it is $this->get(IEventDispatcher::class) that is used